src/results/results.jl

```diff
@@ -105,6 +105,10 @@ function reopt_results(m::JuMP.AbstractModel, p::REoptInputs; _n="")
         add_flexible_hvac_results(m, p, d)
     end
 
+    if !isnothing(p.s.backup_reliability)
```

Comment:
This is always true and unnecessary. I think that we should let the user decide if they want to run the backup reliability calculation the same way we let them decide to run the outage simulator: that is they pass the results dictionary from `run_reopt` or `reopt_results` in to `simulate_outages`.